ilastik/shell/gui/ilastik-style-osx.qss

```diff
@@ -0,0 +1,38 @@
+/*
+ * Adding style sheet for QSlider fixes an osx-specific issue: updating the
+ * handle programmatically via slider.setValue is not reflected in the UI once
+ * the slider has been moved manually.
+ * This behavior is gone after setting the style sheet.
+ * link to Qt issue: https://bugreports.qt.io/browse/QTBUG-96522
+ * ilastik issue: https://github.com/ilastik/ilastik/issues/2623
+ */
+
+QSlider::add-page:horizontal {
+    background: #e0e0e0;
+    border: 1px solid #dadada;
+    border-radius: 2px
+}
+
+QSlider{
+    margin: 2px 2px;
```

Comment:
```suggestion
    margin: 2px;
```
...unless QSS has different semantics than CSS, and this line means something other than "add 2px margin to all four sides".